test/core/xds/xds_bootstrap_test.cc

```diff
@@ -55,12 +55,27 @@ namespace grpc_core {
 namespace testing {
 namespace {
 
+MATCHER_P(EqXdsServer, name, "equals XdsServer") {
+  auto* server = static_cast<const GrpcXdsBootstrap::GrpcXdsServer*>(arg);
+  if (!::testing::ExplainMatchResult(::testing::Not(nullptr), server,
+                                     result_listener) ||
+      !::testing::ExplainMatchResult(::testing::Not(nullptr),
+                                     server->channel_creds_config(),
+                                     result_listener)) {
+    return false;
+  }
+  return ::testing::ExplainMatchResult(name, server->server_uri(),
+                                       result_listener) &&
+         ::testing::ExplainMatchResult(
+             "fake", server->channel_creds_config()->type(), result_listener);
```

Comment:
Instead of hard-coding the value "fake" here, please pass it in as a parameter to the matcher.